libraries/fof/controller/controller.php

```diff
@@ -3347,56 +3347,15 @@ protected function _csrfProtection()
 				break;
 		}
 
-		$hasToken = false;
-		$session  = JFactory::getSession();
+		$token = new \Joomla\Cms\Session\CsrfToken(JFactory::getSession());
```

Comment:
I personally think we do need post-installation (in fact currently part of the struggle is trying to keep the post-installation messages to the essentials and not littering them with stuff every time there is an update).

TFA's in your boat completely